pkg/cloud/openstack/machine/actuator.go

```diff
@@ -18,17 +18,16 @@ package machine
 
 import (
 	"context"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
-	"net"
 	"os"
 	"reflect"
 	"strings"
 	"time"
 
 	"github.com/golang/glog"
+	errorwrapper "github.com/pkg/errors"
```

Comment:
```suggestion
```
Please open an issue to discuss adding a dependency on `github.com/pkg/errors` and using throughout `cluster-api` and `cluster-api-provider-openstack`